eng/common/templates/job/job.yml

```diff
@@ -101,7 +101,7 @@ jobs:
     # handle key-value variable syntax.
     # example:
     # - [key]: [value]
-    - ${{ if and(eq(variable.name, ''), eq(variable.group, '')) }}:
+    - ${{ if and(eq(variable.name, ''), eq(variable.group, ''), eq(variable.template, '')) }}:
```

Comment:
This is an eng/common file which needs to be updated in Arcade: https://github.com/dotnet/arcade/blob/b888df17a4acb65630c1e9ad5e94f22a33b62ab0/eng/common/templates/job/job.yml#L104

Reply:
Arcade PR is here: https://github.com/dotnet/arcade/pull/12524